rpcs3/Emu/Cell/Modules/cellScreenshot.h

```diff
@@ -29,19 +29,38 @@ struct CellScreenShotSetParam
 
 struct screenshot_manager
 {
-	atomic_t<bool> is_enabled{ false };
+	std::mutex mtx;
```

Comment:
Use `shared_mutex` for now, it's universal replacement.

Reply:
done